### PR TITLE
Protect: UI Adjustments

### PR DESCRIPTION
**Pull request by nateweller**
Related to https://github.com/Automattic/jetpack-scan-team/issues/1358

## Proposed changes:

* Hide the "auto-fix" column in the threat history list.
* Hide the threat item footer when it doesn't have any content.
* Use default button size for IP list save buttons.
* Remove the secondary block list description.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/Automattic/jetpack-scan-team/issues/1358

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Review the UI and validate it against the screenshots below.

## Screenshots

Screenshot 2024-08-14 at 6 34 27 PM

Screenshot 2024-08-14 at 6 34 36 PM

Screenshot 2024-08-14 at 6 53 36 PM

### CODE REVIEW

**Review by dkmyta — DISMISSED** [review]
Works great!

A minor things surfaced while reviewing this that I am thinking we could quickly fit into this PR if you agree. When you initiate unignoring a threat when you do not yet have a full user connection, the connection gate suggests that you need to connect to ignore or fix, but mentions nothing about unignoring:
Screen Shot on 2024-08-16 at 06-15-53 (1)

**Comment by dkmyta** [comment]
Oh! One other thing just came to mind, we have a different display for Auto-fix section in mobile view. Right now it looks like these changes remove it entirely, which is what we want for History, but I think we do want it to remain for the Scanner view:
Screen Shot on 2024-08-16 at 11-02-38

**Comment by nateweller** [comment]
@dkmyta Thanks for the review! Ran a script which should fix the tests.

> A minor things surfaced while reviewing this that I am thinking we could quickly fit into this PR if you agree. When you initiate unignoring a threat when you do not yet have a full user connection, the connection gate suggests that you need to connect to ignore or fix, but mentions nothing about unignoring:

I'm thinking we can leave this as-is for now, I'm just concerned about the copy getting too complicated and thinking that the action of un-ignoring would be covered under the umbrella of "ignoring" in this copy, if that makes any sense? 😄

Otherwise if we updated the modal to show a specific feature description based on the action, i.e. not list everything a user connection is needed for, that would be more ideal IMO but maybe out of the scope of this one.

File: projects/plugins/protect/src/js/components/threats-list/paid-list.jsx [hunk 1]
```diff
@@ -135,39 +137,41 @@ const ThreatAccordionItem = ( {
...
-				) }
-				{ 'current' === status && (
-					<>
+			{ [ 'ignored', 'current' ].includes( status ) && (
+				<div className={ styles[ 'threat-footer' ] }>
+					{ 'ignored' === status && (
 						<Button
 							isDestructive={ true }
 							variant="secondary"
 							onClick={ handleIgnoreThreatClick() }
```
Comment (nateweller) [inline projects/plugins/protect/src/js/components/threats-list/paid-list.jsx]:
```suggestion
							onClick={ handleUnignoreThreatClick() }
```

**Review by dkmyta — APPROVED** [review]
Good to go!

**Pull request merged**